src/ContentClient.ts

```diff
@@ -1,30 +1,30 @@
 import { hashV0, hashV1 } from '@dcl/hashing'
 import { Entity, EntityType } from '@dcl/schemas'
-import { Fetcher, mergeRequestOptions, RequestOptions, retry } from 'dcl-catalyst-commons'
+import { IFetchComponent } from '@well-known-components/http-server'
```

Comment:
I'd advise to not include WKC in this lib, it will poison all react apps with @types/node and hundreds of unwanted dependencies.